scene/resources/visual_shader_sdf_nodes.h

```diff
@@ -49,6 +49,8 @@ class VisualShaderNodeSDFToScreenUV : public VisualShaderNode {
 
 	virtual String generate_code(Shader::Mode p_mode, VisualShader::Type p_type, int p_id, const String *p_input_vars, const String *p_output_vars, bool p_for_preview = false) const override;
 
+	virtual Category get_category() const override { return CATEGORY_TEXTURES; }
```

Comment:
I'm unsure exactly what category to assign these to

Reply:
We might want to add a new category for SDF nodes in the future. Or we just use CATEGORY_TEXTURES or CATEGORY_SPECIAL, at least that's what I would do for now.